src/audio/pipeline.c

```diff
@@ -229,8 +229,10 @@ int pipeline_free(struct pipeline *p)
 	}
 
```

Comment:
typo: s/Memoery/memory.Other than that it looks good to me.

Reply:
Updated